cdc/entry/mounter.go

```diff
@@ -532,17 +532,6 @@ func (m *mounter) mountRowKVEntry(tableInfo *model.TableInfo, row *rowKVEntry, d
 			}
 			corrupted = true
 		}
-
-		// NOTICE: When the old Value feature is off,
-		// the Delete event only needs to keep the handle key column.
-		if row.Delete && !m.enableOldValue {
```

Comment:
Can we get rid of this `enableOldValue` field from mounter?

Reply:
no, it still used as `fillWithDefaultValue`